### ISSUE TITLE
REC task SFT result

### ISSUE THREAD

**Issue by cosmos1030** [om-ai-lab/VLM-R1#311]
Thank you for providing this amazing repository.
I followed the instructions to perform SFT on the REC task and found that the results on the Lisa dataset were significantly better than those reported here.
After just 500 training steps, we achieved over 58% accuracy, which is noticeably higher than the reported 55.xx%.
After 15,000 training steps, the model performed even better, reaching over 65% accuracy on the Lisa grounding dataset.
I didn’t change anything from the default LLaMAFactory settings.
Has anyone else observed similar results?

**Comment by Magic-Pony** [comment]
Me too. 100 steps, 60.62, 600 steps 60.01 for SFT on LISA. Although slightly worse than my RL result, it is much better than the report claims.

**Comment by VickyYOX** [comment]
May I ask if only cross entropy loss is used during SFT using LLaMa-Factory? Wouldn't it be not enough for training REC task?
